packages/wallets/unsafe-burner/src/adapter.ts

```diff
@@ -41,11 +42,12 @@ export class UnsafeBurnerWalletAdapter extends BaseSignerWalletAdapter {
     }
 
     get readyState() {
-        return WalletReadyState.Installed;
+        return this._readyState;
     }
 
     async connect(): Promise<void> {
         this._keypair = new Keypair();
+        this._readyState = WalletReadyState.Installed;
```

Comment:
Not that it matters too much, but changing this shouldn't be needed. Loadable adapters typically just stay marked loadable even after they are loaded. We also aren't firing a `readyStateChange` event when it changes, so the React context won't find out about it.

Reply:
Yeah. I can't remember why I did this. I'll revert it and maybe we'll find out.